Installation/Mapping/vrijbrp.zgwZaakEigenschapToFirstRegistration.mapping.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "zgwZaakToFirstRegistration",
+  "$id": "https://vrijbrp.nl/mapping/vrijbrp.zgwZaakToFirstRegistration.mapping.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Mapping.schema.json",
+  "version": "0.0.1",
+  "passTrough": false,
+  "mapping": {
+    "referentienummer": "zaak.identificatie",
+    "geslacht": "rolValues.betrokkeneIdentificatie.geslachtsaanduiding",
+    "voornamen": "rolValues.betrokkeneIdentificatie.voornamen",
+    "voorvoegsel": "rolValues.betrokkeneIdentificatie.voorvoegselGeslachtsnaam",
+    "achternaam": "rolValues.betrokkeneIdentificatie.geslachtsnaam",
+    "geboortedatum": "rolValues.betrokkeneIdentificatie.geboortedatum",
+    "geboorteplaats": "zaakEigenschapValues.inp&#46;geboorteplaats",
+    "geboorteland": "zaakEigenschapValues.geboorteland",
+    "land_van_herkomst": "zaakEigenschapValues.land_van_herkomst",
+    "e-mail": "zaakEigenschapValues.sub&#46;emailadres",
+    "telefoon": "zaakEigenschapValues.telefoonBuitenland",
+    "postcode": "zaakEigenschapValues.aoa&#46;postcode",
+    "huisnummer": "zaakEigenschapValues.aoa&#46;huisnummer",
+    "huisletter": "zaakEigenschapValues.wpl&#46;huisletter",
+    "toevoeging": "zaakEigenschapValues.wpl&#46;huisnummertoevoeging",
+    "straatnaam": "zaakEigenschapValues.gor&#46;openbareRuimteNaam",
+    "woonplaats": "zaakEigenschapValues.wpl&#46;woonplaatsNaam",
+    "datum ingang bewoning": "zaakEigenschapValues.datumVerhuizing",
+    "opmerkingen": "zaak.toelichting"
+  },
+  "cast": {
+    "referentienummer": "unsetIfValue==zaak.identificatie",
+    "geslacht": "unsetIfValue==rolValues.betrokkeneIdentificatie.geslachtsaanduiding",
+    "voornamen": "unsetIfValue==rolValues.betrokkeneIdentificatie.voornamen",
+    "voorvoegsel": "unsetIfValue==rolValues.betrokkeneIdentificatie.voorvoegselGeslachtsnaam",
+    "achternaam": "unsetIfValue==rolValues.betrokkeneIdentificatie.geslachtsnaam",
+    "geboortedatum": "unsetIfValue==rolValues.betrokkeneIdentificatie.geboortedatum",
+    "geboorteplaats": "unsetIfValue==zaakEigenschapValues.inp&#46;geboorteplaats",
+    "geboorteland": "unsetIfValue==zaakEigenschapValues.geboorteland",
+    "land van vorig adres": "unsetIfValue==zaakEigenschapValues.geboorteland",
```

Comment:
dont^ see this one in mapping, do we need to cast this?

Reply:
This seems to me like a preparation for later changes. It won't work here now, but that is caught in the service. If SIM decides they can use separate values, then we can use this.

Reply:
As long as it doesn't break anything 👍